packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/HazardButton/types.ts

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export interface HarzardButtonProps {
+  type?: 'button' | 'reset' | 'submit';
+  title?: string;
+  color?: string;
+  hover?: string;
+  onClick?: () => {};
```

Comment:
@SaraVieira I think it's better to return a `void` type here

```suggestion
  onClick?: () => void;
```